confidence: 18
seconds_per_order: 10
appid: 85c03cfb
